cli/tests/integration/repl_tests.rs

```diff
@@ -623,6 +623,54 @@ fn missing_deno_dir() {
   assert!(err.is_empty());
 }
 
+#[test]
+fn custom_history_path() {
+  use std::fs::{read, remove_file};
+  const DENO_REPL_HISTORY: &str = "custom_history.txt";
+  let test_deno_repl_history =
+    test_util::testdata_path().join(DENO_REPL_HISTORY);
```

Comment:
Instead of using the testdata directory, we should use a temporary directory (search for `let temp_dir = TempDir::new();` for an example in this file)

Reply:
Makes sense. I based that approach on the `missing_deno_dir` test in the same file. Perhaps it should be changed as well?

Reply:
Yes, that would be better.